.travis.yml

```diff
@@ -1,28 +1,36 @@
 language: python
+
 python:
 - '3.6'
 - '3.5'
 - '3.4'
 
 matrix:
   include:
-    - python: 3.6
-      sudo: required
-      dist: trusty
+    - python: '3.4'
       env:
-      - TOXENV=style
+      - DNF=yum OS=centos7
```

Comment:
It is not used, it's there only to be visible in the Travis build table, so the CentOS build will be easily distinguished when looking at it.